docs/_docs/reference/metaprogramming/macros.md

```diff
@@ -192,7 +185,7 @@ quote but no splice between the parameter binding of `T` and its
 usage. But the code can be rewritten by adding an explicit binding of a `Type[T]`:
 
 ```scala
-def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T])(using Type[R], Quotes): Expr[T => R] =
+def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T], Type[R])(using Quotes): Expr[T => R] =
```

Comment:
can't mix anon and named using parameters

Reply:
```suggestion
def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T], r: Type[R])(using Quotes): Expr[T => R] =
```